sources.cfg

```diff
@@ -3,3 +3,8 @@ extends = http://plonesource.org/sources.cfg
 extensions = mr.developer
 
 auto-checkout =
+    ftw.book
+
+
+[branches]
+ftw.book = jone-dexterity
```

Comment:
Yes, you are kind of right.
This is relevant for running the tests on the CI against the development branch of `ftw.book` in order to prove that it will work together.
Before merging this PR, we need to finish the development in `ftw.book` and cut a new release in order to define such a constraint.